packages/universal-components/src/Button/ButtonInner.js

```diff
@@ -52,29 +79,37 @@ export default function ButtonInner({
     testID = `ButtonInner${testIDProps}`;
   }
 
+  const padding = getPadding(leftIcon, rightIcon);
```

Comment:
It was slightly confusing for me at first glance. Because I was expecting actual padding value.
Probably something like `paddingType/Token` can be more suitable.

Reply:
Sorry for poor naming, will change it now

Reply:
Done!